tools/probe/probe_vop_contexts.py

```
@@ -0,0 +1,180 @@
+            self.contexts["attribvop (SOP)"] = geo.createNode(
+                "attribvop", run_init_scripts=True
+            )
+            self.contexts["volumevop (SOP)"] = geo.createNode(
+                "volumevop", run_init_scripts=True
+            )
+        except Exception as e:
+            logger.error(f"SOP context setup failed: {e}")
+
+        try:
+            self.contexts["matnet (Shading)"] = hou.node("/mat")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)